src/main/java/org/osc/sdk/controller/api/SdnRedirectionApi.java

```diff
@@ -94,11 +95,11 @@ String installInspectionHook(NetworkElement networkElem, InspectionPortElement i
      * No-op if no inspection hook is found on SDN controller to be deleted.
      * </p>
      *
-     * @param networkElem  provides the network element containing protected ports
+     * @param networkElements  provides a list of network elements containing protected ports
```

Comment:
same as above : add comments about what inspectionport argument holds and also what do we have in List of networkElements